### PR TITLE
Run rubocop and clean up

### PR DESCRIPTION
**Pull request by mullermp**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -22,7 +24,7 @@ end
 
 # Ensure the test:integration task behaves as it always has
 desc 'run integration tests'
-task 'test:integration' do |t|
+task 'test:integration' do |_t|
```
Comment (alextwoods) [inline Rakefile:27]: Can just remove the argument entirely here?

File: aws-record.gemspec [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  spec.authors       = ["Amazon Web Services"]
-  spec.email         = ["mamuller@amazon.com", "alexwoo@amazon.com"]
-  spec.summary       = "AWS Record library for Amazon DynamoDB"
-  spec.description   = "Provides an object mapping abstraction for Amazon DynamoDB."
-  spec.homepage      = "https://github.com/aws/aws-sdk-ruby-record"
-  spec.license       = "Apache 2.0"
+  spec.name          = 'aws-record'
+  spec.version       = File.read(File.expand_path('VERSION', __dir__)).strip
+  spec.authors       = ['Amazon Web Services']
+  spec.email         = ['mamuller@amazon.com', 'alexwoo@amazon.com']
```
Comment (jterapin) [inline aws-record.gemspec:7]: where's my email 🥺

**Review by alextwoods — APPROVED** [review]
Nice - LGTM

File: aws-record.gemspec (hunk 2 above)
Comment (mullermp) [reply aws-record.gemspec:7]: Haha, sorry, good catch, I'm going to replace all of this with `aws-dr-rubygems@amazon.com`

File: Rakefile (hunk 1 above)
Comment (mullermp) [reply Rakefile:27]: Yep

**Pull request merged**